packages/augur-ui/src/modules/common/icons.tsx

```diff
@@ -582,23 +582,11 @@ export const DownloadIcon = (
 );
 
 export const DoubleArrowIcon = (
-  <svg viewBox="0 0 15 16">
-    <path
-      d="M9.92676 9L13.1575 12L9.92676 15"
-      strokeWidth="2"
-      strokeMiterlimit="10"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-    />
-    <path d="M12 12C12 12.5523 11.5523 13 11 13H0.999998C0.447714 13 -1.43051e-06 12.5523 -1.43051e-06 12C-1.43051e-06 11.4477 0.447714 11 0.999999 11L11 11C11.5523 11 12 11.4477 12 12Z" />
-    <path
-      d="M4.23242 1L1.00165 4L4.23242 7"
-      strokeWidth="2"
-      strokeMiterlimit="10"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-    />
-    <path d="M2.15918 4C2.15918 4.55228 2.60689 5 3.15918 5H13.1592C13.7115 5 14.1592 4.55228 14.1592 4C14.1592 3.44772 13.7115 3 13.1592 3L3.15918 3C2.60689 3 2.15918 3.44772 2.15918 4Z" />
+  <svg viewBox="0 0 13 14" fill="none">
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M4.07327 6.9408C4.32532 6.67241 4.31208 6.2505 4.04369 5.99845L1.64051 3.74156L4.04369 1.48466C4.31208 1.23261 4.32532 0.810708 4.07327 0.542318C3.82122 0.273928 3.39931 0.260685 3.13092 0.512737L0.210285 3.25559C0.0761067 3.3816 0 3.55749 0 3.74156C0 3.92563 0.0761067 4.10151 0.210285 4.22752L3.13092 6.97038C3.39931 7.22243 3.82122 7.20919 4.07327 6.9408Z" fill="#0E0E21"/>
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M8.08842 13.6075C7.83636 13.3391 7.84961 12.9172 8.118 12.6651L10.5212 10.4082L8.118 8.15133C7.84961 7.89928 7.83636 7.47738 8.08842 7.20899C8.34047 6.9406 8.76237 6.92735 9.03076 7.1794L11.9514 9.92226C12.0856 10.0483 12.1617 10.2242 12.1617 10.4082C12.1617 10.5923 12.0856 10.7682 11.9514 10.8942L9.03076 13.637C8.76237 13.8891 8.34047 13.8759 8.08842 13.6075Z" fill="#0E0E21"/>
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M1.33333 3.66537C1.33333 3.29718 1.63181 2.9987 2 2.9987H11.3333C11.7015 2.9987 12 3.29718 12 3.66537C12 4.03356 11.7015 4.33203 11.3333 4.33203H2C1.63181 4.33203 1.33333 4.03356 1.33333 3.66537Z" fill="#0E0E21"/>
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M1.33333 10.332C1.33333 9.96384 1.63181 9.66537 2 9.66537H11.3333C11.7015 9.66537 12 9.96384 12 10.332C12 10.7002 11.7015 10.9987 11.3333 10.9987H2C1.63181 10.9987 1.33333 10.7002 1.33333 10.332Z" fill="#0E0E21"/>
```

Comment:
remove fill colors, `fill-rule` and `clip-rule` throws errors, must be `fillRule` and `clipRule` in react.